public_html/assets/css/nf-core.css

```diff
@@ -106,13 +100,12 @@ img.emoji{
   }
 }
 
-.toasts-container {
-    position: fixed;
-    bottom: 5px;
-    right: 5px;
-}
 .toast {
-  background-color: rgba(255,255,255,.90)
+  background-color: rgba(255,255,255,.90);
+  position: fixed;
+  bottom: 3rem;
+  right: 3rem;
```

Comment:
```suggestion
  right: 1rem;
```

Reply:
Putting it a bit lower into the corner
<img width="1611" alt="Screenshot 2020-07-30 at 10 57 26" src="https://user-images.githubusercontent.com/6169021/88905664-cd345200-d256-11ea-9658-d4af82a3b40f.png">